test/eth.accounts.signTransaction.js

```diff
@@ -614,6 +614,7 @@ var tests = [
             chainId: 1,
             nonce: 0,
             gas: 27200,
+            maxPriorityFeePerGas: '0x3B9ACA00',
```

Comment:
This shouldn't be necessary, was there a reason why it was added?

Reply:
The original fixture was using this as default value, so to generate identical hashes have to use the previous value of `maxPriorityFeePerGas`

Reply:
Oh I see, thank you